### DISCUSSION TITLE
Can we use a black pill?

### DISCUSSION THREAD

**Discussion by uzair-io** [erichelgeson/BlueSCSI-v1#11] [General]
Is it possible to use a STM32 STM32F411 ‘Black Pill’ for this project?

**Comment by erichelgeson** [comment]
In it's current state - no it is for the stm32f103. I know a few were looking at cross compiling it. If you get it working let us know!

**Reply by mg-man** [reply]
It looks like TbA has it working...

**Reply by uzair-io** [reply]
any details?

**Reply by erichelgeson** [reply]
You can find them here - https://androda.work/shop/
